content/en/glossary/terms/rule.md

```diff
@@ -0,0 +1,11 @@
+---
+title: rule
+synonyms:
+  - detection rule
+  - compliance rule
+core_product:
+  - security
+  - cspm
+---
+
+A rule evaluates the configuration of a resource to validate an element related to one or more controls. Rules may map to multiple controls, requirements, and frameworks.
```

Comment:
Since we're putting this in the main glossary, would it be appropriate to add "security" somewhere in here to give people context? Maybe "A security rule evaluates..."